src/Monolog/Formatter/ElasticaFormatter.php

```diff
@@ -72,7 +72,7 @@ protected function getDocument(array $record): Document
     {
         $document = new Document();
         $document->setData($record);
-        $document->setType($this->type);
+        $document->setOpType($this->type);
```

Comment:
The `opType` is the operation type and not at all the same as `types` in ES<7. See https://github.com/ruflin/Elastica/blob/b1b6c980cce2ffc3f278e0d3d02e61b174cee52e/src/AbstractUpdateAction.php#L186